nengo_fpga/networks/fpga_pes_ensemble_network.py

```diff
@@ -63,6 +63,9 @@ class FpgaPesEnsembleNetwork(nengo.Network):
         that is used to handle UDP communication between the host PC and the
         FPGA board. The full list of parameters can be found here:
         https://github.com/nengo/nengo-fpga/blob/master/nengo_fpga/sockets.py#L425
+    recurrence : float or (D_out, D_in) array_like, optional
+        Defines the transform for a recurrent connection. If ``None``, no
+        recurrent connection will be built.
```

Comment:
I'm still wondering if the feedback connection should have a default synapse.... Maybe @tbekolay or @drasmuss will have other opinions about this. 
Options:
- Have a default synapse and put it in the docs somewhere
- Dont have a default synapse (synapse=None), throw an error (the standard Nengo error?) if the user tries to build the model without setting the synapse.

Reply:
My opinion was to have a default since the synapse is _not_ part of the constructor. For me, it seems odd to throw an error for not explicitly setting an attribute post-construction... that being said I will certainly yield to my software overlords if they have strong opinions :pray:

Reply:
Would definitely have a default synapse; like @bmorcos says, you shouldn't be able to get into a situation where you get an error for making something and not knowing to set some magical attribute.

If you want to be more explicit, you can add the feedback connection as a method (`add_feedback(..., synapse=nengo.Lowpass(0.1)`) which you call in `__init__` when you pass in something for `recurrence`. Then if people need the advanced options they call the method themselves. Just a half-formed idea, at the very least you should have a default synapse.